docs/reference/modules/remote-clusters.asciidoc

```diff
@@ -0,0 +1,217 @@
+[[modules-remote-clusters]]
+== Remote clusters
+
+ifndef::include-xpack[]
+The _remote clusters_ module allows establishing uni-directional connections to
+a remote cluster. This functionality is used in
+<<modules-cross-cluster-search,cross-cluster search>>.
+endif::[]
+ifdef::include-xpack[]
+The _remote clusters_ module allows establishing uni-directional connections to
+a remote cluster. This functionality is used in cross-cluster replication,
+<<modules-cross-cluster-search,cross-cluster search>>.
+endif::[]
+
+Remote cluster connections work by configuring a remote cluster and connecting
+only to a limited number of nodes in the remote cluster. Each remote cluster is
+referenced by a name and a list of seed nodes.  When a remote cluster is
+registered, its cluster state is retrieved from one of the seed nodes so that up
+to 3 _gateway nodes_ are selected to be connected to as part of remote cluster
```

Comment:
it might be preexisting but we should make clear that this is by default up to 3. it sounds like a max.